tests/ci/style_check.py

```diff
@@ -68,17 +72,88 @@ def process_result(result_folder):
         return state, description, test_results, additional_files
 
 
+def parse_args():
+    parser = argparse.ArgumentParser("Check and report style issues in the repository")
+    parser.add_argument("--push", default=True, help=argparse.SUPPRESS)
+    parser.add_argument(
+        "--no-push",
+        action="store_false",
+        dest="push",
+        help="do not commit and push automatic fixes",
+        default=argparse.SUPPRESS,
+    )
+    return parser.parse_args()
+
+
+def checkout_head(pr_info: PRInfo):
+    # It works ONLY for PRs, and only over ssh, so either
+    # ROBOT_CLICKHOUSE_SSH_KEY should be set or ssh-agent should work
+    assert pr_info.number
+    if not pr_info.head_name == pr_info.base_name:
```

Comment:
Just out of curiosity, this should not be required, since github secrets (`ROBOT_CLICKHOUSE_SSH_KEY`) is not available for forks, right?

Reply:
Sure, but then the script will crash